src/parser/core/modules/EventEmitter.ts

```diff
@@ -325,6 +325,11 @@ class EventEmitter extends Module {
       );
     }
 
+    if (event.type === 'create') {
```

Comment:
Not sure if we still need this console event haha

Reply:
Definitely not lol